mappings/net/minecraft/entity/damage/DamageTracker.mapping

```diff
@@ -9,6 +9,7 @@ CLASS net/minecraft/class_1283 net/minecraft/entity/damage/DamageTracker
 	FIELD field_5877 entity Lnet/minecraft/class_1309;
 	METHOD <init> (Lnet/minecraft/class_1309;)V
 		ARG 1 entity
+	METHOD method_33937 getBiggestAttackerId ()I
```

Comment:
Might want to document this is getting the biggest attacker's id, not the biggest id of attackers